cmd/overlay.go

```diff
@@ -30,7 +31,7 @@ var overlayCmd = &model.CommandGroup{
 	Usage:    "overlay",
 	Short:    "Work with OpenAPI Overlays",
 	Long:     utils.RenderMarkdown(overlayLong),
-	Commands: []model.Command{overlayCompareCmd, overlayValidateCmd, overlayApplyCmd},
+	Commands: []model.Command{overlayCompareCmd, overlayValidateCmd, overlayApplyCmd, overlayApplyMultipleCmd},
```

Comment:
This is a non breaking change with pure overlay apply. Hence IMO this would be cleaner to keep as one command and push the capability to handle multiple overlays there.